sql/src/test/java/io/crate/analyze/GroupByAnalyzerTest.java

```diff
@@ -344,13 +345,23 @@ public void testGroupWithInvalidNullLiteral() throws Exception {
     @Test
     public void testGroupByHaving() throws Exception {
         AnalyzedRelation relation = analyze("select sum(floats) from users group by name having name like 'Slartibart%'");
-        assertThat(relation.having().query(), isFunction("op_like"));
+        assertThat(relation.having().query(), isFunction(LikeOperators.OP_LIKE));
         Function havingFunction = (Function) relation.having().query();
         assertThat(havingFunction.arguments().size(), is(2));
         assertThat(havingFunction.arguments().get(0), isReference("name"));
         assertThat(havingFunction.arguments().get(1), isLiteral("Slartibart%"));
     }
 
+    @Test
+    public void testGroupByHaving2() throws Exception {
```

Comment:
Can we have more verbose test names that hint at the concrete scenario being tested?